main.c

```diff
@@ -218,6 +231,54 @@ static const struct wl_keyboard_listener keyboard_listener = {
 	.modifiers = noop,
 };
 
+static void touch_handle_down(void *data, struct wl_touch *touch, uint32_t serial,
+                          uint32_t time, struct wl_surface *surface, int32_t id,
```

Comment:
Style: we don't align, we use two tabs instead.

Reply:
Done 1e21b359fb7530f226e4c0e5d75f8d6a5befaabd.